include/deal.II/fe/fe_values.h

```diff
@@ -313,6 +369,24 @@ namespace FEValuesViews
                                  std::vector<typename ProductType<gradient_type,typename InputVector::value_type>::type> &gradients) const;
 
     /**
+     * Same as above, but using a of vector local degree-of-freedom values.
+     *
+     * The DoF values typically would be obtained in the following way:
```

Comment:
I'm working on a fix using doxygen's `@copydoc` tag. I'll then only put one of these paragraphs per view task - I think that's a fair compromise.